server/models/user.model.ts

```diff
@@ -13,16 +13,16 @@ export default class User extends BaseModel {
 
   static get relationMappings() {
     return {
-      associated_companies: {
+      companiesHandling: {
```

Comment:
Can we rename this? Maybe managedCompanies? Or at least a comment here is valuable.